### DISCUSSION TITLE
My account got flagged?

### DISCUSSION THREAD

**Discussion by theonlywes** [community/community#136506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Reposting this on a different account because of this whole ordeal I realized no one could even see the post somehow:

Apparently my account got "flagged". I received no notice of this or even a reason why. I only found out because a friend said my GitHub account didn't exist. Apparently it was only hidden from public view. Furthermore, I don't even have any type of banner like I've seen on other accounts when they get flagged. I have contacted support with no response for a week now. I can't even delete my account to recreate it. I am also the owner of a few organizations that had projects that me and a few other people were developing. Still, even those organizations were not spared. There is not public access to them and I can't delete, rename or transfer ownership on any of them. I can still login and use my account normally but everything is hidden for public view. This is very frustrating as I was trying to build a reputation on GitHub to show to my future employers. I still don't know what I did wrong and can only guess. Is there any other steps I can take to get this resolved? Thanks.

I'm also now beginning to wonder if no one can see my support ticket in the same way that no one can see the community post I made.

**Comment by NicholasGannon** [comment]
Just recently dealt with this - I think that Github has been dealing with an extra amount of support tickets lately, so it might take a while for you to get a response. I would recommend doing the reinstatement ticket if you haven't already done so, and you should be set once they see your ticket - Hope this helps!

**Reply by theonlywes** [reply]
Thanks for the advice. I already filled out a ticket. Were you able to recover your account?

**Reply by NicholasGannon** [reply]
Yes, I was - It just took a while. I've heard that there can be a wide range of time in terms of how long it takes for support to respond.

**Comment by theonlywes** [comment]
Well, support just got back to me, they said I violated the usage terms for codespaces and won't un flagged my account. (I think I may have left a misconfigured program running that was using more resources than it should have.) However, I still think it's a little harsh to disable my whole account. I'm not sure what else I can do. I am asking them if I can at least transfer ownership of my organizations and I'll see what they say.

**Reply by NicholasGannon** [reply]
I'm sorry to hear that - I definitely would recommend reaching out to them again, and see if you can get that ownership transfer.

**Comment by wiedu-admin** [comment]
I am also the owner of the organization account and have submitted a reinstatement ticket, continuously updating GitHub Support with explanations. 

Unfortunately, it has now been 11 days without any response. Meanwhile, over 160 accounts in my organization are still unable to use the service properly, yet we are still being charged for GitHub Copilot Seats. 

This situation is very frustrating, and I hope @github @github-staff can assist us in restoring our accounts to normal as soon as possible or provide a solution to resolve this issue.
